students/JeroenSweerts/Lesson4/dict_lab.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#########################Dictionaries 1#########################################
+#Create a dictionary containing “name”, “city”, and “cake” for “Chris” from
+#“Seattle” who likes “Chocolate” (so the keys should be: “name”, etc,
+#and values: “Chris”, etc.)
+d = {}
+d['name'] = 'Chris'
+d['city'] = 'Seattle'
+d['cake'] = 'Chocolate'
```

Comment:
Unless there's a good reason, the 'dictionary literal' syntax is more conventional in Python that initializing and adding to an empty dict:
```
d = {"name": "Chris",
     "city": "Seattle",
     "cake": "Chocolate"}
```